src/test/resources/emails/sessionUnpublishedEmailForInstructor.html

```diff
@@ -0,0 +1,16 @@
+Hello Instructor1 Course1,
+<p/>
+The email below has been sent to students of course: idOfTypicalCourse1.<p/><br><br>=== Email message as seen by the students ===<br>
```

Comment:
Perhaps add a line breaks after each `<br>` tag? That way, the code will look closer to the final output.